docs/source/developer/setup.rst

```diff
@@ -150,6 +150,96 @@ be able to use it with openwisp-radius (and openwisp-radius) easily:
     ./manage.py migrate --fake openwisp-radius 0001_initial_freeradius
     ./manage.py migrate
 
+Automated periodic tasks
+------------------------
+
+Some periodic commands are necessary to enable certain
+features and facilitate database cleanup.
+There are two ways to automate these tasks:
```

Comment:
let's clarify that these steps are needed for production usage, in development and testing this is not really required